src/main/java/org/iban4j/CountryCode.java

```diff
@@ -766,41 +300,24 @@
      */
     private final String alpha3;
 
-    private CountryCode(final String name, final String alpha3) {
-        this.name = name;
-        this.alpha3 = alpha3;
-    }
-
-
     /**
-     * Get the country name.
-     *
-     * @return The country name.
+     * SEPA identifier
      */
-    public String getName() {
-        return name;
-    }
+    private final String sepaIdentifier;
```

Comment:
Why don't you use a boolean type for this field ?

Reply:
Well, no special reason. We got used to use literals in many places in our application, so this was just straightforward decision.  It just had to do it's job ... :-)
If you guys vote for bool, I'll refactor ...